src/MovieDetails.js

```diff
@@ -1,36 +1,75 @@
-import React from 'react'
+import React, {Component} from 'react'
 import './MovieDetails.css'
-import {movieDeets} from './movieData'
 
-function MovieDetails() {
+class MovieDetails extends Component {
+  constructor() {
+    super()
+    this.state = {
+       singleMovieDetails: {},
+       singleMovieVideo: {}
+    }
+  }
 
-  console.log(movieDeets)
-  const backgroundImage = {
-    backgroundImage: 'url(https://image.tmdb.org/t/p/original//oazPqs1z78LcIOFslbKtJLGlueo.jpg)'
+  componentDidMount = () => {
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${this.props.id}`)
+      .then(response => response.json())
+      .then(data => this.setState({ singleMovieDetails: data.movie }))
+      .catch(error => alert(`Error: ${error}`))
   }
 
-  return (
-    <main className='movie-details' style={backgroundImage}>
-      <div className='movie-box'>
-        <section className='video-box'>Video Box</section>
-        <section className='details-box'>
-          <h2>Title: {movieDeets.movie.title}</h2>
-          <p className='overview'>Overview: {movieDeets.movie.overview}</p>
-          <div className='detail-columns'>
-            <div className='column'>
-              <p>Genre: {movieDeets.movie.genres}</p>
-              <p>Budget: {movieDeets.movie.budget}</p>
-              <p>Revenue: {movieDeets.movie.revenue}</p>
-            </div>
-            <div className='column'>
-              <p>Runtime: {movieDeets.movie.runtime} minutes</p>
-              <p>Tagline: {movieDeets.movie.tagline}</p>
+  render() {
+
+    const { singleMovieDetails } = this.state
+
+    const backgroundImage = {
+      backgroundImage: `url(${singleMovieDetails.backdrop_path})`,
+      backgroundSize: 'cover'
+    }
+
+    return (
+      <main className='movie-details' style={backgroundImage}>
+        <button className='close-button' onClick={() => this.props.handleClose()}>X</button>
+        <div className='movie-box'>
+          <section className='details-box'>
+            <h2>Title: {singleMovieDetails.title}</h2>
+            <p className='overview'>Overview: {singleMovieDetails.overview}</p>
+            <div className='detail-columns'>
+              <div className='column'>
+                <p>Genre: {singleMovieDetails.genres}</p>
+                <p>Budget: {singleMovieDetails.budget}</p>
+                <p>Revenue: {singleMovieDetails.revenue}</p>
+              </div>
+              <div className='column'>
+                <p>Runtime: {singleMovieDetails.runtime} minutes</p>
+                <p>Tagline: {singleMovieDetails.tagline}</p>
+              </div>
             </div>
-          </div>
-        </section>
-      </div>
-    </main>
-  )
+          </section>
+        </div>
+      </main>
+    )
+  }
 }
 
 export default MovieDetails;
+
+//VIDEO START
+
+// {/* <video width="320" height="240" className='video-box' controls>
```

Comment:
Love that you're trying to get the video to play in here as well.  Super cool! 🎉